### ISSUE TITLE
A prefix with a dot inside messes up the path names

### ISSUE THREAD

**Issue by UnderTheCarpet** [fkjogu/stochasticsampling#43]
In GitLab by @fkjogu on Oct 10, 2017, 10:57

If a prefix contains a dot, everything after the dot is chopped of.

Example
```
/foo/kap20.0_sa-2.0-2017-10-09_164002_v0_4_2-c7e2a7b/kap20.0_sa-2.msgpack.lzma
```
instead of
```
/foo/kap20.0_sa-2.0-2017-10-09_164002_v0_4_2-c7e2a7b/kap20.0_sa-2.0-2017-10-09_164002_v0_4_2-c7e2a7b.msgpack.lzma
```

**Comment by UnderTheCarpet** [comment]
In GitLab by @fkjogu on Oct 10, 2017, 10:59

changed the description

**Comment by UnderTheCarpet** [comment]
In GitLab by @fkjogu on Oct 10, 2017, 11:08

closed via commit ba15a5a7d85143d71a4c13d73f540c1cddf7fa1d
